packages/math-input/src/components/keypad/pre-algebra-page.js

```diff
@@ -14,7 +14,7 @@ type Props = {|
     onClickKey: (keyConfig: string) => void,
 |};
 
-export default class NumericInputPage extends React.Component<Props> {
+export default class PreAlgebraPage extends React.Component<Props> {
```

Comment:
The good news is that because this is a default export, its name really doesn't matter. The importer defines whatever name it wants for the default export when imported. 

See: https://github.com/Khan/perseus/blob/2f028befc0a81f99262f66d2f34407580538e191/packages/math-input/src/components/keypad/index.js#L8